### DISCUSSION TITLE
Add the default name tags of sensors to the documentation (and also how proto and the APIs relate)

### DISCUSSION THREAD

**Discussion by toshiharutf** [cyberbotics/webots#5449] [Q&A]
Hello,
First of all, I want to congratulate the Webots team for this nice simulator. I just been using it for a couple of days and I found it much more intuitive and faster to grasp than other sims.
I have a question regarding the default name tags of sensors (and perhaps other components) when they are added in the proto file without a name tag. I've been searching for info regarding this, but I got no luck.

For example

```
a_sample_proto.proto

Robot{
...
Accelerometer{}
...
InertialUnit{}
...
}
```
By checking the provided controller examples I was able to understand that the default tag name for the InertialUnit sensor is "inertial unit", and for the Accelerometer is "accelerometer"

A controller example:
```
WbDeviceTag imu = wb_robot_get_device("inertial unit");
WbDeviceTag acc = wb_robot_get_device("accelerometer");
```

Are these "default tag names" mentioned somewhere? I could add them to the documentation, but I would need a source where they are defined.

Additionally, as someone who is just starting to use Webots, it would be nice to know the relationship of the proto file and the APIs. I could also add some of this info in the docs if the team think is a good idea.

Best regards,

**Comment by omichel** [comment]
Good point. This is not documented on the device pages such as https://www.cyberbotics.com/doc/reference/accelerometer
The information can be found however in the node definition files in https://github.com/cyberbotics/webots/tree/master/resources/nodes
But I agree, this is not very user friend and we should document in the device pages because this "name" field has a different default value than the node it derives from (e.g., the [Solid](https://www.cyberbotics.com/doc/reference/solid) node). So, it would be good to specify it in the derived classes.
That's something easy to do if you would like to contribute: Simply edit the documentation files and open a PR that we will review and eventually merge. The link to propose a documentation contribution is available on the top of each documentation page under "Found an error?[ Contribute on GitHub!](https://github.com/cyberbotics/webots/edit/released/docs/reference/accelerometer.md)"
@toshiharutf: would you be willing to do this? We will be happy to help you if needed.

**Reply by toshiharutf** [reply]
Hi Michael,
Thanks for the info and happy to help.
I would make a first one as format prototype for the team to check. Once approved, we can use it as a template for the other ones.
Best regards,
